ui: remove disk read and write time graphs

vilterp opened the pull request:
The values are suspect (we've seen >>1 disk second per second on machines
with one disk), and it seems that what people really want to see is
latency of individual disk operations, anyway.

Not generally cool to remove time series, but these have only been available in betas.

Release note (admin ui change): Remove read and write graphs from the
hardware dashboard.

Commit:
ui: remove disk read and write time graphs

The values are suspect (we've seen >>1 disk second per second on machines
with one disk), and it seems that what people really want to see is
latency of individual disk operations, anyway.

Release note (admin ui change): Remove read and write graphs from the
hardware dashboard.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/30488)
<!-- Reviewable:end -->

Commit:
server: remove disk read and write time metrics

It's unclear what these mean, their values don't make much sense, and
they're not what people need during perf debugging -- latency of
individual disk ops is more important.

Release note: None

couchand reviewed (approved):
> Not generally cool to remove time series

OTOH if we don't really trust it that seems like a good idea

vilterp commented:
bors r+
